client/app/controllers/application.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    actions:{
+        saveGameInParent(outcome, rank) {
```

Comment:
Instead of *InParent, you could use onSave, onUpdate, and onDelete as this seems to be an event listener design